jasypt-spring-boot/src/main/java/com/ulisesbocchio/jasyptspringboot/properties/JasyptEncryptorConfigurationProperties.java

```diff
@@ -1,13 +1,13 @@
 package com.ulisesbocchio.jasyptspringboot.properties;
 
+import java.util.List;
+
 import com.ulisesbocchio.jasyptspringboot.EncryptablePropertyFilter;
 import com.ulisesbocchio.jasyptspringboot.util.AsymmetricCryptography.KeyFormat;
 import lombok.Data;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.boot.context.properties.NestedConfigurationProperty;
 
-import java.util.List;
-
```

Comment:
No need to move the List import.

Reply:
Looks good. I'm gonna try to release this upcoming weekend. Can you create a PR for the samples repo too if any of the tests break?

Reply:
@j0rdanit0 - agreed! Reverted.

Reply:
@ulisesbocchio yes no problem. I'll have a look at this now and confirm when done.

Reply:
@ulisesbocchio samples repo is currently using version 2.1.2 of jasypt-spring-boot (Line 19 in parent pom.xml). Are you happy for my PR to bump this to 2.1.3-SNAPSHOT?

Reply:
yeah, that's fine. Thanks!

Reply:
@ulisesbocchio PR is here: https://github.com/ulisesbocchio/jasypt-spring-boot-samples/pull/9